packages/react-server-dom-webpack/src/__tests__/ReactFlightDOMEdge-test.js

```diff
@@ -543,6 +543,55 @@ describe('ReactFlightDOMEdge', () => {
     expect(await iterator.next()).toEqual({value: undefined, done: true});
   });
 
+  // @gate enableFlightReadableStream
+  it('dedupes objects inside async iterables', async () => {
```

Comment:
I'm confused. This is a new test but it asserts that there isn't deduping when the test describes deduping. Can we just gate that it fails until we address the underlying limitation? maybe I am misunderstanding something

Reply:
We often just assert on the broken behavior inline with a comment. However, since this entire test is broken I'll name it different.